Microsoft.Health.Fhir.Ingest.sln

```diff
@@ -71,6 +71,10 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "healthkitOnFhir", "healthki
 		sample\templates\healthkitOnFhir\fhirmapping.json = sample\templates\healthkitOnFhir\fhirmapping.json
 	EndProjectSection
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Microsoft.Health.Fhir.Ingest.Common", "src\lib\Microsoft.Health.Fhir.Ingest.Common\Microsoft.Health.Fhir.Ingest.Common.csproj", "{37F9E4E3-851E-4B8A-BECA-FD4388679E22}"

```

Comment:
Mentioned in other comment but I believe Microsoft.Health.Fhir.Ingest.Schema is better fit.  We already have several common libraries and another could be confusing.  Plus I don't think we should have anything in this library except schemas going forward.

Reply:
Cool, thanks for the suggestion. Agree that the Schema would be better, I will update.